ha/fault_tolerance/event_manager.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+'''Module whoch provides interfaces to subscribe or unsubscribe for HA
+   events to cortx components'''
+
```

Comment:
Please add one integration or unit test
which can be provided to component to know how to use api

Reply:
Agree

Reply:
Added as a TODO for now.